curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-014.md

```diff
@@ -19,10 +19,16 @@ Use a single type selector to center the `h1`, `h2` and `p` elements at the same
 
 # --hints--
 
-Test 1
+You should use a single type selector for all three elements.
 
 ```js
+assert(code.match(/(h1|h2|p), (h1|h2|p), (h1|h2|p) {/))
```

Comment:
Should we check for closing brackets?

Reply:
Thoughts on an approach? I'm thinking, based on the order of the tests, I should add `.*}` and allow anything between the braces for this step.

Reply:
that makes sense.

Reply:
I see you point. this could also match } of the next selector. I think it is fine as it is